test/views/CheckYourAnswersPageSpec.scala

```diff
@@ -61,13 +61,13 @@ class CheckYourAnswersPageSpec extends SpecBase with ViewBehaviours {
       Selectors.appealDetailsH2 -> h2AppealDetails,
       Selectors.descriptionListDtKey(1) -> "Key",
       Selectors.descriptionListDdValue(1) -> "Value",
-      Selectors.descriptionListDdAction(1) -> "Change",
+      Selectors.descriptionListDdAction(1) -> "Change answer for ’Key’",
```

Comment:
I'm not sure how I feel about this, I would have thought we'd have one check for the visual text and another check for the visually hidden text? So we know that the visual content is correct and the visually hidden content is correct separately?

Reply:
done